python_exercises/exercises/helpers.py

```diff
@@ -0,0 +1,89 @@
+from collections import namedtuple
+from time import sleep
+
+def ennumerate_task_list(task_list):
```

Comment:
nitpick: there is a typo in ennumerate -> enumerate. There are a few other variables/functions with the same typo.